openstack/neutron/Chart.yaml

```diff
@@ -7,17 +7,17 @@ dependencies:
   - condition: mariadb.enabled
     name: mariadb
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.14.2
+    version: 0.18.2
   - name: memcached
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.5.3
+    version: 0.6.8
   - condition: mariadb.enabled
     name: mysql_metrics
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.3.6
+    version: 0.4.2
   - name: rabbitmq
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.11.1
+    version: 0.16.0
```

Comment:
rabbitmq chart has been updated to 0.17.1 recently with user-credential-updater sidecar, it would be better to use here the latest version